client/src/component/Main.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Switch, Route } from 'react-router-dom' //Switch and Route Library
+import Home from '../pages/home/home.jsx'
+import SignUp from '../pages/signUp/signUp.jsx'	//link to Sign Up page
+import loginRoute from '../pages/login/loginRoute.jsx' //link to Router for login
+
+function Main() {
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class Welcome {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
Hi @adriandarian, this suggestion will not change anything as you need to extend `Component` in a class for this to work.

Also I always suggest to keep it as simple as possible first, then add complexity when needed